frontend/src/services/GroupMemberService.js

```diff
@@ -16,4 +16,7 @@ export default {
   cancelDependent(request) {
     return apiRequest().then((axiosInstance) => axiosInstance.post(resources.groupMember.cancelDependent, request))
   },
+  reinstateOverAgeDependent(request) {
+    return apiRequest().then((axiosInstance) => axiosInstance.post(resources.groupMember.cancelDependent, request))
+  },
```

Comment:
Was this accidently added to the wrong file?

Reply:
Wrong file :).